media_driver/linux/common/codec/ddi/media_libva_encoder.cpp

```diff
@@ -378,8 +380,11 @@ VAStatus DdiEncode_CreateContext(
 
     MOS_STATUS eStatus = pCodecHal->Allocate(encCtx->m_encode->m_codechalSettings);
 
-#ifdef _MMC_SUPPORTED
+    //Get the priority from VAAPI.
     PMOS_INTERFACE osInterface = pCodecHal->GetOsInterface();
+    osInterface->pfnSetGpuPriority(osInterface, priority);
```

Comment:
Please check osInterface is available before using it.

Reply:
Done